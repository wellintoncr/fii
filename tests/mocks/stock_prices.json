{
    "Meta Data": {
        "1. Information": "Monthly Prices (open, high, low, close) and Volumes",
        "2. Symbol": "NCHB11.SAO",
        "3. Last Refreshed": "2022-11-14",
        "4. Time Zone": "US/Eastern"
    },
    "Monthly Time Series": {
        "2022-11-14": {
            "1. open": "93.2000",
            "2. high": "94.8000",
            "3. low": "90.1200",
            "4. close": "91.4800",
            "5. volume": "62776"
        },
        "2022-10-31": {
            "1. open": "91.9700",
            "2. high": "94.8300",
            "3. low": "91.5500",
            "4. close": "93.2000",
            "5. volume": "89489"
        },
        "2022-09-30": {
            "1. open": "91.5200",
            "2. high": "95.5000",
            "3. low": "90.9900",
            "4. close": "91.5200",
            "5. volume": "102551"
        },
        "2022-08-31": {
            "1. open": "94.7600",
            "2. high": "94.7600",
            "3. low": "90.3400",
            "4. close": "91.5100",
            "5. volume": "124497"
        },
        "2022-07-29": {
            "1. open": "95.7500",
            "2. high": "98.6400",
            "3. low": "93.9100",
            "4. close": "94.2800",
            "5. volume": "126078"
        },
        "2022-06-30": {
            "1. open": "96.2000",
            "2. high": "97.9700",
            "3. low": "94.8300",
            "4. close": "95.7500",
            "5. volume": "107421"
        },
        "2022-05-31": {
            "1. open": "100.9900",
            "2. high": "100.9900",
            "3. low": "95.2000",
            "4. close": "96.5000",
            "5. volume": "180906"
        },
        "2022-04-29": {
            "1. open": "94.4000",
            "2. high": "100.9800",
            "3. low": "94.0900",
            "4. close": "99.9900",
            "5. volume": "153050"
        },
        "2022-03-31": {
            "1. open": "92.1900",
            "2. high": "95.0000",
            "3. low": "91.5500",
            "4. close": "94.1000",
            "5. volume": "141182"
        },
        "2022-02-25": {
            "1. open": "91.9100",
            "2. high": "94.2400",
            "3. low": "90.9800",
            "4. close": "92.3000",
            "5. volume": "127919"
        },
        "2022-01-31": {
            "1. open": "91.0100",
            "2. high": "94.9900",
            "3. low": "90.4700",
            "4. close": "91.7000",
            "5. volume": "128849"
        },
        "2021-12-30": {
            "1. open": "90.2000",
            "2. high": "92.5000",
            "3. low": "89.1100",
            "4. close": "90.8800",
            "5. volume": "132937"
        },
        "2021-11-30": {
            "1. open": "91.6500",
            "2. high": "92.8800",
            "3. low": "88.9100",
            "4. close": "89.9000",
            "5. volume": "119620"
        },
        "2021-10-29": {
            "1. open": "94.3300",
            "2. high": "94.9800",
            "3. low": "90.6100",
            "4. close": "91.6800",
            "5. volume": "96147"
        },
        "2021-09-30": {
            "1. open": "92.0100",
            "2. high": "95.3900",
            "3. low": "90.8900",
            "4. close": "94.9900",
            "5. volume": "106326"
        },
        "2021-08-31": {
            "1. open": "94.0000",
            "2. high": "95.3000",
            "3. low": "90.5000",
            "4. close": "92.0000",
            "5. volume": "73622"
        },
        "2021-07-16": {
            "1. open": "93.9100",
            "2. high": "95.9600",
            "3. low": "92.8000",
            "4. close": "95.7800",
            "5. volume": "38523"
        },
        "2021-06-30": {
            "1. open": "95.0000",
            "2. high": "96.9700",
            "3. low": "89.0000",
            "4. close": "93.9000",
            "5. volume": "104091"
        },
        "2021-05-31": {
            "1. open": "98.7300",
            "2. high": "99.1000",
            "3. low": "94.0000",
            "4. close": "94.0300",
            "5. volume": "101590"
        },
        "2021-04-30": {
            "1. open": "104.0000",
            "2. high": "107.0000",
            "3. low": "98.3500",
            "4. close": "99.1800",
            "5. volume": "97557"
        },
        "2021-03-31": {
            "1. open": "98.5000",
            "2. high": "105.0000",
            "3. low": "96.1600",
            "4. close": "104.7000",
            "5. volume": "127084"
        },
        "2021-02-26": {
            "1. open": "103.8000",
            "2. high": "108.8000",
            "3. low": "98.2900",
            "4. close": "98.5000",
            "5. volume": "141355"
        },
        "2021-01-29": {
            "1. open": "102.4900",
            "2. high": "106.9500",
            "3. low": "95.0000",
            "4. close": "103.8000",
            "5. volume": "140770"
        },
        "2020-12-30": {
            "1. open": "93.2200",
            "2. high": "105.0000",
            "3. low": "92.0000",
            "4. close": "98.5000",
            "5. volume": "61569"
        },
        "2020-11-30": {
            "1. open": "91.6100",
            "2. high": "94.3500",
            "3. low": "88.1000",
            "4. close": "92.1300",
            "5. volume": "22859"
        },
        "2020-10-30": {
            "1. open": "94.3400",
            "2. high": "94.4000",
            "3. low": "85.0100",
            "4. close": "91.5900",
            "5. volume": "73840"
        }
    }
}