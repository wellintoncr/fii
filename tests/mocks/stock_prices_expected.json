{
    "name": "SMTH11",
    "stock_prices": {
        "2022-10-31": {
            "open": "91.9700",
            "high": "94.8300",
            "low": "91.5500",
            "close": "93.2000",
            "volume": "89489"
        },
        "2022-09-30": {
            "open": "91.5200",
            "high": "95.5000",
            "low": "90.9900",
            "close": "91.5200",
            "volume": "102551"
        },
        "2022-08-31": {
            "open": "94.7600",
            "high": "94.7600",
            "low": "90.3400",
            "close": "91.5100",
            "volume": "124497"
        },
        "2022-07-29": {
            "open": "95.7500",
            "high": "98.6400",
            "low": "93.9100",
            "close": "94.2800",
            "volume": "126078"
        },
        "2022-06-30": {
            "open": "96.2000",
            "high": "97.9700",
            "low": "94.8300",
            "close": "95.7500",
            "volume": "107421"
        },
        "2022-05-31": {
            "open": "100.9900",
            "high": "100.9900",
            "low": "95.2000",
            "close": "96.5000",
            "volume": "180906"
        },
        "2022-04-29": {
            "open": "94.4000",
            "high": "100.9800",
            "low": "94.0900",
            "close": "99.9900",
            "volume": "153050"
        },
        "2022-03-31": {
            "open": "92.1900",
            "high": "95.0000",
            "low": "91.5500",
            "close": "94.1000",
            "volume": "141182"
        },
        "2022-02-25": {
            "open": "91.9100",
            "high": "94.2400",
            "low": "90.9800",
            "close": "92.3000",
            "volume": "127919"
        },
        "2022-01-31": {
            "open": "91.0100",
            "high": "94.9900",
            "low": "90.4700",
            "close": "91.7000",
            "volume": "128849"
        },
        "2021-12-30": {
            "open": "90.2000",
            "high": "92.5000",
            "low": "89.1100",
            "close": "90.8800",
            "volume": "132937"
        },
        "2021-11-30": {
            "open": "91.6500",
            "high": "92.8800",
            "low": "88.9100",
            "close": "89.9000",
            "volume": "119620"
        },
        "2021-10-29": {
            "open": "94.3300",
            "high": "94.9800",
            "low": "90.6100",
            "close": "91.6800",
            "volume": "96147"
        },
        "2021-09-30": {
            "open": "92.0100",
            "high": "95.3900",
            "low": "90.8900",
            "close": "94.9900",
            "volume": "106326"
        },
        "2021-08-31": {
            "open": "94.0000",
            "high": "95.3000",
            "low": "90.5000",
            "close": "92.0000",
            "volume": "73622"
        },
        "2021-07-16": {
            "open": "93.9100",
            "high": "95.9600",
            "low": "92.8000",
            "close": "95.7800",
            "volume": "38523"
        },
        "2021-06-30": {
            "open": "95.0000",
            "high": "96.9700",
            "low": "89.0000",
            "close": "93.9000",
            "volume": "104091"
        },
        "2021-05-31": {
            "open": "98.7300",
            "high": "99.1000",
            "low": "94.0000",
            "close": "94.0300",
            "volume": "101590"
        },
        "2021-04-30": {
            "open": "104.0000",
            "high": "107.0000",
            "low": "98.3500",
            "close": "99.1800",
            "volume": "97557"
        },
        "2021-03-31": {
            "open": "98.5000",
            "high": "105.0000",
            "low": "96.1600",
            "close": "104.7000",
            "volume": "127084"
        },
        "2021-02-26": {
            "open": "103.8000",
            "high": "108.8000",
            "low": "98.2900",
            "close": "98.5000",
            "volume": "141355"
        },
        "2021-01-29": {
            "open": "102.4900",
            "high": "106.9500",
            "low": "95.0000",
            "close": "103.8000",
            "volume": "140770"
        },
        "2020-12-30": {
            "open": "93.2200",
            "high": "105.0000",
            "low": "92.0000",
            "close": "98.5000",
            "volume": "61569"
        },
        "2020-11-30": {
            "open": "91.6100",
            "high": "94.3500",
            "low": "88.1000",
            "close": "92.1300",
            "volume": "22859"
        },
        "2020-10-30": {
            "open": "94.3400",
            "high": "94.4000",
            "low": "85.0100",
            "close": "91.5900",
            "volume": "73840"
        }
    }
}